core/src/actions/swagger.ts

```diff
@@ -120,12 +138,17 @@ export class Swagger extends Action {
         title: parentPackageJSON.name,
         license: { name: parentPackageJSON.license },
       },
-      host: url.replace("http://", "").replace("https://", ""),
-      basePath: `/api/v${API_VERSION}`,
-      tags: tags.map((tag) => {
-        return { name: tag };
-      }),
-      schemes: url.includes("https") ? ["https"] : ["http"],
+      host: process.env.WEB_URL
+        ? process.env.WEB_URL.replace("https://", "").replace("http://", "")
+        : "localhost",
+      basePath: `/api/${API_VERSION}`,
```

Comment:
Does this change what swagger thinks our API endpoints are? (`basePath` changed `/v1/`->`/1/`)

Reply:
Simplified and cleaned up in 686fe9b2a8aeb3c2d8a4fb00d19746e6c78dac55